client/view/index/Index.html

```diff
@@ -133,13 +145,46 @@ <h2>Try an example deck</h2>
 
 export default {
 	data() {
+		var url = window.location.href.split("?");
+		if (url[1] !== undefined) window.location.replace("/#/home");
+		var searchParams = new URLSearchParams(url[1]);
```

Comment:
Components should never touch the browser location directly - if a component wants to change the url, they should call `stateRouter.go` via the `stateGo` mediator function.  If a component wants to get information from the url, [the `resolve` function can pass it in via the `parameters` argument](https://github.com/TehShrike/abstract-state-router/#resolvedata-parameters-callbackerr-contentredirectstatename-stateparameters).